src/models/connectionProfile.ts

```diff
@@ -127,7 +127,9 @@ export class ConnectionProfile extends ConnectionCredentials implements IConnect
 			});
 
 		return prompter.prompt(questions, true).then(async answers => {
-			if (answers.authenticationType === 'AzureMFA') {
+			if(!answers){
+				return undefined;
+			} else if (answers?.authenticationType === 'AzureMFA') {
```

Comment:
Saw an error in debug console when a user would exit out of `Create Connection Profile` while filling out the info. Therefore the `answers` would be undefined. We should handle this properly to result in an undefined instead, possibly could throw an error as well that user cancelled.

Reply:
Can you please do this in a separate PR? That way it can be reviewed much quicker and won't clutter up this PR